chat-plugins/scavengers.js

```diff
@@ -144,13 +195,86 @@ exports.commands = {
 		scavengers.answers = null;
 		scavengers.participants = {};
 		scavengers.finished = [];
+		scavengers.updatinglb = false;
+		scavengers.numblitzes = 0;
 		if (scavengers.result) {
 			Rooms('scavengers').addRaw('<div class="broadcast-blue"><strong>' + scavengers.result + '</strong></div>');
 		} else {
 			Rooms('scavengers').addRaw('<div class="broadcast-blue"><strong>The Scavenger Hunt was reset by <em>' + Chat.escapeHTML(user.name) + '</em>.</strong></div>');
 		}
 		scavengers.result = null;
 	},
+	scavtop: 'scavengerstop',
+	scavengerstop: function (target, room, user) {
+		if (room.id !== 'scavengers') return this.errorReply('This command can only be used in the Scavengers room.');
```

Comment:
Use quote marks for these kind of responses unless you are (and have a reason to) use template strings instead